test/fixtures/guess-hash-seed.js

```diff
@@ -134,7 +134,7 @@ const slow_str_gen = (function*() {
   let strgen_i = 0;
   outer:
   while (1) {
-    const str = '#' + (strgen_i++);
+    const str = `# ${strgen_i++}`;
```

Comment:
The space after `#` should be removed.

Reply:
Incorporated the mentioned change. PTAL. Many Thanks.